woocommerce/woocommerce-android
# [Better Products Tablets] Reordering and deleting from the products images screen doesn't work

Closes: #11182 
Closes: #11181


### Description

The PR fixes a bug when the toolbar menu was not updated after a state change

### Testing instructions

* Products -> Product with images
* Rearrange the images
* Notice "check" on the toolbar
* That saves the change
Do the same with the removal of a picture.

### Images/gif




https://github.com/woocommerce/woocommerce-android/assets/4923871/88a29f3f-6983-47cd-bfd8-4f227663d199
